src/app/pages/shop-page/components/products/products-form/products-form.component.html

```diff
@@ -1,47 +1,43 @@
-<form [formGroup]="productForm" (ngSubmit)="submit()">
+<form [formGroup]="productForm" (ngSubmit)="submit()" (keydown.enter)="$event.preventDefault()">
```

Comment:
I'm not sure you need this `keydown.enter` if you use Reactive Forms. What happens otherwise?

Reply:
Because im using (ngSubmit) =submit() it fires and closes dialog even if form is invalid